WIZARD_UIFILES/upgrade_uifile_ger

```diff
@@ -21,7 +21,7 @@
     "step_title": "Adminport",
     "items": [{
         "type": "textfield",
-        "desc": "W&auml;hle einen HTTP Port f&uuml; openhab:",
+        "desc": "W&auml;hle den die Ports f&uuml;r Openhab aus:",
```

Comment:
W&auml;hle die Ports

Reply:
I believe the attempt is to identify HTTP/HTTPS ports; Shouldn't it also not mention HTTP here? And shouldn't line 43 perhaps use this 'wahle die ports ..' text?

Reply:
Hey pieterzen, 
I have take a Picture from the Wizard. 

![image](https://user-images.githubusercontent.com/16503629/46524770-1ea56d00-c88a-11e8-8a60-9e859b6492ce.png)

Reply:
I see; So removing HTTP is sufficient. Thanks for feedback